src/components/checkbox-group/checkbox/bl-checkbox.css

```diff
@@ -0,0 +1,67 @@
+:host {
+    display: inline-block;
+
+    --bg-color: var(--bl-checkbox-bg-color, var(--bl-color-primary));
+    --bg-color-disabled: var(--bl-color-tertiary, var(--bl-color-disabled));
```

Comment:
Actually, all those variables don't change by any prop or attributes, we can set them directly. We don't have to define another variable. You can set it like this:

```
    width: var(---bl-size-m);
    height: var(--bl-size-m);
```